### PR TITLE
Update TransactionPreview modal

### PR DESCRIPTION
**Pull request by buberdds**
Minor UI to make content more readable in extension 
- split content into sections so it's easier to read amount
- unify modal titles
- hash takes 100% width

current ext
Screenshot 2022-11-03 at 11 23 40

updated
Screenshot 2022-11-03 at 11 10 21

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -260,20 +260,18 @@
...
       "signing": "Signing transaction",
       "submitting": "Submitting transaction"
     },
     "types": {
-      "addEscrow": "Delegating your tokens to a validator and generate rewards",
-      "reclaimEscrow": "Reclaiming your tokens delegated to a validator",
-      "transfer": "Transferring tokens from your account to another"
+      "addEscrow": "Confirm delegation of your tokens to a validator",
+      "reclaimEscrow": "Confirm reclaim of your tokens delegated to a validator",
+      "transfer": "Confirm transfer of tokens from your account to another"
```
Comment (lukaw3d) [inline src/locales/en/translation.json:274]: Wording of these isn't as clear as before, and I liked the "Preview transaction" title :cry:

File: src/app/components/ResponsiveGridRow/index.tsx [hunk 2]
```diff
@@ -1,18 +1,43 @@
...
 import React, { memo } from 'react'
+import styled from 'styled-components'
+import { normalizeColor } from 'grommet/utils'
+
+interface StyledResponsiveGridRowValueProps extends BoxProps {
+  withSeparator?: boolean
+}
+
+const StyledResponsiveGridRowValue = styled(Box)<StyledResponsiveGridRowValueProps>`
+  @media only screen and (max-width: ${({ theme }) => `${theme.global?.breakpoints?.small?.value}px`}) {
```
Comment (lukaw3d) [inline src/app/components/ResponsiveGridRow/index.tsx:11]: Might make sense to display separator on all screen sizes

File: src/app/components/TransactionModal/index.tsx [hunk 3]
```diff
@@ -49,26 +50,28 @@ export function TransactionModal() {
...
+              <TransactionPreview
+                chainContext={chainContext}
+                preview={preview}
+                walletAddress={walletAddress!}
+              />
+            </>
           )}
           {step === TransactionStep.Preview && (
-            <Box direction="row" gap="small" alignSelf="end" pad={{ top: 'large' }}>
+            <Box direction="row" gap="small" justify="between" pad={{ top: 'large' }}>
```
Comment (lukaw3d) [inline src/app/components/TransactionModal/index.tsx:74]: Is spreading the buttons a decision from some mocks?

File: src/locales/en/translation.json (hunk 1 above)
Comment (buberdds) [reply src/locales/en/translation.json:274]: Ok, I will gather more feedback around this one.

File: src/app/components/TransactionModal/index.tsx (hunk 3 above)
Comment (buberdds) [reply src/app/components/TransactionModal/index.tsx:74]: Based on feedback from Don, our UX pattern for confirmation modal action buttons should be: 
- on desktop buttons are aligned to right
- on mobile: space-between/cancel on left, confirm on right

File: src/locales/en/translation.json (hunk 1 above)
Comment (buberdds) [reply src/locales/en/translation.json:274]: UX update: unify it all into one: "Confirm transaction"

Comment (lukaw3d) [reply src/locales/en/translation.json]: Did you mean to bring back transaction type after unifying the title?

Comment (lukaw3d) [reply src/locales/en/translation.json]: That removes the most vital information from the confirmation dialog

| Before | After |
| --- | --- |
| a | b |
| c | d |

Comment (buberdds) [reply src/locales/en/translation.json]: In that case let's keep current modal

**Pull request closed without merging**